### PR TITLE
Rename SNOW example

### PR DESCRIPTION
**Pull request by agrimmer**
Rename severity_based_on_impact.yaml to ServiceNow_calculate_severity_based_on_impact.yaml

### CODE REVIEW

**Review by amohrd — APPROVED** [review]

**Comment by agardnerIT** [comment]
This has one approving review but merging is blocked until is has 2.
Repo has no `CODEOWNERS` so who else can approve?

**Review by wlfghdr — APPROVED** [review]

**Pull request merged**
